siege_utilities/integrations/parsons/_auth.py

```
@@ -0,0 +1,213 @@
+
+
+# ---------------------------------------------------------------------------
+# Bridge
+# ---------------------------------------------------------------------------
+
+
+def bridge_credentials(
+    connector: str,
+    *,
+    profile: str | None = None,
+    manager: Any | None = None,
```

Ruff (0.16.1)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `manager`

(ANN401)